packages/xo-web/src/common/intl/messages.js

```diff
@@ -790,6 +790,7 @@ const messages = {
   memoryHostState:
     'RAM: {memoryUsed} used on {memoryTotal} ({memoryFree} free)',
   hardwareHostSettingsLabel: 'Hardware',
+  hyperThreading: 'Hyperthreading',
```

Comment:
`'Hyper-threading'`

Reply:
not `Hyper-Threading` ? with `T` ?

Reply:
Here's how @olivierlambert wants it https://github.com/vatesfr/xen-orchestra/issues/4262#issuecomment-496965914

Reply:
Wikipedia says "Hyper-threading" but let's wait for @olivierlambert on that one, I'm not really sure...

Reply:
yeah it sounds to be `Hyper-threading`

Reply:
We might use `Hyper-threading (SMT)` to be completely correct.

Reply:
corrected as requested :heavy_check_mark: